src/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -41,17 +41,9 @@ See the LICENSE file in the project root for more information.
       DependsOnTargets="$(LinkNativeDependsOn)">
 
     <ItemGroup>
-      <NativeLibrary Condition="$(NativeCodeGen) == ''" Include="$(IlcPath)/sdk/libbootstrapper.a" />
```

Comment:
The files under BuildIntegration are meant to be used in other projects - see https://github.com/dotnet/corert/blob/master/Documentation/how-to-build-and-run-ilcompiler-in-console-shell-prompt.md. This change breaks these steps.